### ISSUE TITLE
More detail + deep dive + search on /admin/queue 

### ISSUE THREAD

**Issue by realkinetix** [friendica/friendica#9580]
As per the discussion at https://forum.friendi.ca/redir/43?url=https://horche.demkontinuum.de/display/2196d4ee-165f-ad24-2d00-59f517882855 I mentioned it could be nice to have some enhancements made to the queue listing:

a) paginate so it's not just a hard limit of the oldest 999 entries
b) have a 'retried/retries' column to show how many times that item has been processed
c) show the task expiry date
d) be able to select the task ID to show all details around what the task / parameters are, and then possibly 
e) a search function to do a search of all the data in the tasks and parameters in case we want to see if an instance or user or whatnot is undeliverable for some reason, etc.

I believe that providing additional details to the instance admin would be helpful for understanding what's sitting around and why, and if we can deep dive in to the tasks or gather bunches of them via search, we should be able to see issues reaching certain contacts or instances.  Right now, there's really little detail provided, and at least in Frio, we only get 999 listed as a maximum, period.  I have over 60,000 in the queue right now and the only way to find out more is to go diving through the database.

**Comment by AlfredSK** [comment]
Yep. I just wanted to open an issue for the worker queue details. In the current state it is nearly useless. Most entries don't show any meaningful information for the admin. 

For example 'process queue - some ID - some date'. 

I miss the destination (domain), the kind of delivery, the number of delivery attempts, the user who created the queue entry.

**Comment by MrPetovan** [comment]
Not all these informations are relevant for queued tasks, we would need a custom display per task type, hmmm.

**Comment by annando** [comment]
Just like we are currently calling the `execute` function of a worker class to execute it, we could create another function per worker class to explain the task.

**Comment by MrPetovan** [comment]
Indeed, but it would incur many SELECT queries in this list screens.

**Comment by annando** [comment]
Definitely. But I don't know any other way, since each worker has got individual parameters and only the worker does know what they mean.

**Comment by MrPetovan** [comment]
We should start paginate the list and limit to 20-30 items per page, possibly add filtering by task including a total count for each task.
